applications/dashboard/scss/style-compat.scss

```diff
@@ -1,5 +1,5 @@
-// A style sheet for compatibility with new features.
 // This should always be added after style.css and not be overridden.
+
```

Comment:
If you leave your styles in `_style.css` they can be overwritten. Let's make a new folder "compatibility" and move your styles in there. Include them through `style-compat.scss`